packages/map/src/GMap.ts

```diff
@@ -576,7 +583,11 @@ export class GMap extends HTMLWidget {
                     }
                 });
             } else {
-                console.error("Street View data not found for this location.");
+                if(context.useStatusErrorCallback()){
```

Comment:
No need for `useStatusErrorCallback` just call the default implementation (and move the message in there).